ios-app/YouPlay/Services/PlaylistService/PlaylistServiceImpl/PlaylistServiceImpl.swift

```diff
@@ -93,6 +93,10 @@ class PlaylistServiceImpl: PlaylistService {
     
     // Function to create a new playlist for a given user
     func createPlaylist(uid: String, name: String) async {
+        if name.lowercased() == "Liked Songs".lowercased(){
```

Comment:
Could make this a global constant, that way its consistent here and where the default playlists are defined.